docs/source/conf.py

```diff
@@ -0,0 +1,122 @@
+# MIT License
+#
+# Copyright (c) [2024] [Aalto Electric Drives]
```

Comment:
Is this from Aalto? Do we need to use this or we can get a different license? I thought MIT licenses are more "universal"...

Reply:
The conf.py file is from motulator (almost every line). Apparently, when using the code under MIT licence, the lisence has to be included to the code. 

But I noticed the whole text does not have to be there, only the first section.